web/js/router.js

```diff
@@ -38,6 +39,7 @@ function GanomedeRouter () {
             </Route>
           )
         }
+        <Route path={utils.webPath('/chat')} component={ChatRoom} />
```

Comment:
The usernames to look for should be part of the URL, like in the `users` view.

With something like `/chat/test1,test2` giving the initial value for usernames as `test1` and `test2`.

This way, it'll be possible to create a deep link to a chat between 2 users. For example `test1` reported `test2`, and we want to access the chat between them, we can just have a link to `/chat/test1,test2`.

Of course, it should still be possible to go to `/chat/` and find empty inputs.